contrib/README.md

```diff
@@ -0,0 +1,17 @@
+Contributed Software
```

Comment:
This looks good! Can you please add it to `install-docs` target in the Makefile, so people can find it in /usr/share/doc/newsboat/contrib when they install Newsboat?